python-checks/src/test/resources/checks/hotspots/unsafeHttpMethods/django/views.py

```diff
@@ -49,3 +49,13 @@ def sensitive_other_dec(request): # Noncompliant
 
 def other(request):
   ...
+
+
+from django.urls import path
+
+class SameClassView:
```

Comment:
_nit_ I'm not sure `SameClassView` is crystal clear.

```suggestion
class ClassWithViews:
```
or something similar